src/com/facebook/buck/ide/intellij/DefaultIjLibraryFactory.java

```diff
@@ -154,6 +161,12 @@ public void apply(
           .ifPresent(
               sourcePath -> library.addSourceJars(libraryFactoryResolver.getPath(sourcePath)));
       arg.getJavadocUrl().ifPresent(library::addJavadocUrls);
+
+      Path aarUnpackPath = BuildTargets.getScratchPath(
```

Comment:
Can you refactor this part so that if someone changes either `UnzipAar` or `AndroidPrebuiltAarDescription`, this code would be changed too? Right now the folder name is copied and there is no way detect these dependencies. I suppose you can use buildTarget.withFlavors(AndroidPrebuiltAarDescription.AAR_UNZIP_FLAVOR) and then refactor `UnzipAar` to provide access to unpack path.